x-pack/packages/kbn-elastic-assistant/impl/assistant/helpers.ts

```diff
@@ -81,7 +86,14 @@ export const getBlockBotConversation = (
  */
 export const getDefaultConnector = (
   connectors: AIConnector[] | undefined
-): AIConnector | undefined => (connectors?.length === 1 ? connectors[0] : undefined);
+): AIConnector | undefined => {
```

Comment:
Before we only wanted to select a default connector if only one existed, otherwise let the user choose which connector. Why did this change?

Reply:
I believe this was to streamline UX and guide away from the 'No connector' error being so prevalent. Ideally I think we should have a 'Default connector for new conversations' setting and default to that one like we do with System Prompts.